news/2021-01-14-performance-points-updates.md

```diff
@@ -0,0 +1,108 @@
+
+This meant that 3 misses on a 100 note map was functionally equivalent to 3 misses on a 1000 note map.
+
+As suggested by [**Xexxar**](https://osu.ppy.sh/users/2773526), the following change has been applied:
+
+- The miss penalty curve has been adjusted.
+    + Any number of misses on a map instantly incurs a once-off 3% penalty to total pp awarded for both aim and speed.
+    + An additional reduction is applied on any further misses based on the number of total objects in the map.
+    + Old value: Aim and speed values reduced by ```0.97^x``` where x equals the number of misses.
+
+To help illustrate these changes better, consult the graph image below. The green line refers to the old formula, and the purple line refers to the new one. You can also view this graph live and play around with the settings following [this link](https://www.desmos.com/calculator/rshij9757a).
+
+![](/wiki/shared/news/2021-01-14-performance-points-updates/miss-penalty-curve.png)
+
+This change seeks to penalize misses on short maps more, while significantly reducing the impact of a small number of misses on longer maps. The one-off 3% penalty for a miss provides incentive for players to hone any 1-miss plays they might still make, but otherwise makes them considerably less punishing on longer plays.
+
+## Speed + Accuracy Curve Adjustments
+
+As osu! has evolved over time, accuracy has slowly but steadily fallen to the wayside as a principle component for determining the pp values of modern scores. To illustrate this, 500-700pp scores from many years ago sometimes derived as much as 20-40% of their overall pp purely from the accuracy component alone.
+
+Today, many scores pull less than 20% of their score instead, relying on raw speed as the largest influencing component on how scores are set. This has reduced the meaningful impact of accuracy as a skill considerably, which is a peculiar approach for a rhythm game where accuracy should be king.
+
+As suggested by [**Xexxar**](https://osu.ppy.sh/users/2773526), the following change has been applied:
+
+- A new speed curve has been introduced which factors in Overall Difficulty and accuracy.
+    + New value: ![](/wiki/shared/news/2021-01-14-performance-points-updates/new-speed-curve.png)
+    + Old value: ![](/wiki/shared/news/2021-01-14-performance-points-updates/old-speed-curve.png)
+- Speed value is now scaled with the number of 50s made in a score in order to penalize doubletapping.
+    + If there is less than one 50 per 500 objects in a map, the speed value is reduced by ```.98^x``` where x is *half the number of 50s*.
+    + If there are more than one 50 per 500 objects in a map, the speed value is reduced by ```.98^x``` where x is *the number of 50s minus the half the total number of objects in the map divided by 500*.
```

Comment:
The OP of this pull was not updated to the latest changes; see [this comment](https://github.com/ppy/osu/pull/11175#issuecomment-744631553). This should say something like:

* If there is less than one 50 per 500 objects, the factor is a constant 1 (the speed value is unchanged; technically it's 0.98 to the zeroth power - which is 1)
* If there is more than one 50 per 500 objects, the factor is 0.98 to the power of (50 count - total count / 500).

The desmos link is wrong too, as far as I can tell. Should look something like this (the blue graph on the desmos link in the aforementioned comment looks the same):

![2021-01-12-181459_1526x674_scrot](https://user-images.githubusercontent.com/20418176/104349335-16294b80-5503-11eb-88bc-37829a36705c.png)